### PR TITLE
The page

### PR DESCRIPTION
**Pull request by men32z**
- Project finished
- 3 Tabs, contact, menu and Home,
- dist and src folders
- ES6, webpack used here.
- tabs modules imported to index.js
- UI interface for the page.

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "arrow-parens": "off",
+    "no-return-assign": "off"
+  }
+}
```
Comment (bolah2009) [inline .eslintrc.json]: Please let's use the [**correct config files**](https://github.com/microverseinc/linters-config/tree/master/javascript).

> Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))

File: README.md [hunk 2]
```diff
@@ -1 +1,18 @@
...
+# SOLO Project:
+## Project Specifications:
+
+This project comes after the introduction of the ES6 class keyword, ES6 modules, and webpack. The main goal of this project is for the student to show the understanding of the benefits of modularized code as well as the ability to set up a javascript application made of different modules using webpack.
+
+You can find the original project specification [here](https://www.theodinproject.com/courses/javascript/lessons/restaurant-page)
+
+## Live Version
+
+[here](https://raw.githack.com/men32z/js-restaurant-page/the-page/dist/index.html)
```
Comment (bolah2009) [inline README.md:11]: [Optional]

Remember to update this address before merging. It is currently pointing to the `the-page` branch. The page gets broken if the branch is deleted

**Review by bolah2009 — CHANGES_REQUESTED** [review]
Hi @men32z 

Awesome job with the [app](https://raw.githack.com/men32z/js-restaurant-page/e20578684c9b58102a0bb76727c2f14e1ccbe853/dist/index.html)

Can you please use the correct config and submit another CRR?


#### :recycle:  Make these changes and submit another code review request. 👍🏾

#### Feel free to reach me on slack if you having issues implementing these features. :relaxed: 

Good luck with your next submission. :star2: 

Cheers
[Bola](https://bolabuari.com/)

###### PS: Note that comments tagged as `OPTIONAL` are not part of the minimum requirements to approve this PR

File: .eslintrc.json (hunk 1 above)
Comment (men32z) [reply .eslintrc.json]: Sure, I got the file now from that source and I'll take in mind for next projects

**Review by mcrd25 — APPROVED** [review]
# Changes Approved 🤓 💪 🎊 

Hi @men32z, 👋 

Great job! 👏 
You successfully implemented required changes and may now merge and move unto ⏭ Project. 🎉 

***If you have any questions, now or in the future, do not hesitate to contact me on social media, email or on slack and I will be glad to chat with you.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

**Pull request merged**